pkg/backplane/backplane.go

```
@@ -96,6 +115,73 @@ func (c *ClientImpl) CreateReport(ctx context.Context, clusterId string, summary
+
+	deleteRemediationParams := bpapi.DeleteRemediationParams{
+		RemediationInstanceId: response.JSON200.RemediationInstanceId,
+		ManagingCluster:       nil,
+	}
+	if isManagementCluster {
+		managingCluster := bpapi.DeleteRemediationParamsManagingClusterManagement
+		deleteRemediationParams.ManagingCluster = &managingCluster
+	}
+
+	restConfig := &RestConfig{
+		Config:       *cfg,
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gofmt)